### ISSUE TITLE
JavaScript doesn't support decorators

### ISSUE THREAD

**Issue by garethj2** [denoland/deno#24745]
In Deno, JavaScript decorators aren't supported. Obviously they're supported for TypeScript.

This is pretty annoying when one bundles a Deno project into one large JavaScript file to run.

The workaround is to rename the bundled files to `ts` instead of `js`.

Decorators should be supported in both TypeScript and JavaScript.

**Comment by dsherret** [comment]
Yeah, this is just unimplemented because we're waiting for v8 to support it ("Support in pure JavaScript is waiting on implementation in V8." https://deno.com/blog/v1.40#decorators).

**Comment by bartlomieju** [comment]
Relevant issue in V8 bug tracker: https://issues.chromium.org/issues/42202709
